packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -191,26 +192,17 @@ MeetingInfoUtil.getDestinationType = async (from) => {
       return Promise.resolve(options);
     });
   } else if (hydraId.room) {
-    options.type = _CONVERSATION_URL_;
-    try {
-      await webex.internal.services.waitForCatalog('postauth');
-
-      const conversationUrl = webex.internal.conversation.getUrlFromClusterId({
-        cluster: hydraId.cluster,
-        id: hydraId.destination,
-      });
-
-      options.destination = conversationUrl;
-    } catch (e) {
-      LoggerProxy.logger.error(`Meeting-info:util#getDestinationType --> ${e}`);
-      throw e;
-    }
+    LoggerProxy.logger.error(
+      `Meeting-info:util#getDestinationType --> Using the space ID as a destination is no longer supported. Please refer this documentation https://github.com/webex/webex-js-sdk/wiki/Migration-guide-for-USM-meeting to migrate to use the meeting ID or SIP address.`
+    );
+    // Error code 101 added as Space ID deprecated as of beta, Please refer migration guide.
```

Comment:
Do we need this comment? We have a log above that says the same thing right?

Reply:
@sreenara, correct, the above logger code is clear, just adding comments can make it even more understandable, especially for other developers, we can remove if not required.